### PR TITLE
Fixing positioning issue

### PR DESCRIPTION
**Pull request by vbichkovsky**
When there are some repeat-x images in the sprite and some other images have position set to 100%, there is a gap between sprite's right side and images with position = 100% (because width-fix was applied after they had been positioned).
I tried to figure out how to write a test for such case and I must say that your tests are difficult to comprehend ;)

### CODE REVIEW

**Comment by chriseppstein** [comment]
@scottdavis can you review and merge when convenient?

File: lib/compass/sass_extensions/sprites/sprite_methods.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'rational'
```
Comment (scottdavis) [inline lib/compass/sass_extensions/sprites/sprite_methods.rb:1]: Do we need this require?

Comment (vbichkovsky) [inline lib/compass/sass_extensions/sprites/sprite_methods.rb:1]: Nope, it just happened that my commit contained these changes and I created a pull request from it. You only need to change layout_methods.rb

Comment (scottdavis) [inline lib/compass/sass_extensions/sprites/sprite_methods.rb:1]: manually changing something kinda makes using pull requests pointless =p

**Comment by chriseppstein** [comment]
@scottdavis what's the status here? Are you holding this up so that a single require can be removed? >_<

**Comment by scottdavis** [comment]
No I'm holding it up because I'm busy I'll get to it this week I'm going to cherry-pick it into master

**Comment by scottdavis** [comment]
merged 5e42b8f57d885fa603c07908a272003c020d3d9b

**Pull request closed without merging**